content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -195,6 +197,21 @@ To encrypt the data:
              timeout: 3s
    ```
 
+Setting `--encryption-provider-config-automatic-reload` to `true` collapses all health checks to a single health check endpoint. Individual health checks are only available when KMS v1 providers are in use and the encryption config is not auto-reloaded.
+
+Following table summarizes the health check endpoints for each KMS version:
+
+| KMS configurations        | Without Automatic Reload           | With Automatic Reload  |
+| ------------------------- |------------------------------------| -----------------------|
+| KMS v1 only               | Individual Healthchecks            | Single Healthcheck     |
+| KMS v2 only               | Single Healthcheck                 | Single Healthcheck     |
+| Both KMS v1 and v2        | Individual Healthchecks            | Single Healthcheck     |
+| No KMS                    | None                               | Single Healthcheck     |
```

Comment:
With no kms configurations, there is no health check endpoint for kms right?

Reply:
`/healthz/kms-providers` is always present when automatic reload is enabled because you could add a KMS provider at any time.